### PR TITLE
[WIP] Adding CI

### PR DESCRIPTION
**Pull request by junaruga**
DO NOT merge this PR yet.
This is working PR for https://github.com/multiarch/busybox/issues/2 .

Current CI result:
https://travis-ci.org/multiarch/busybox/builds/543095257

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -1,83 +1,88 @@
...
-		ln -s /bin/busybox "${module}"
-	    done
-	)
+        pushd "${arch}/slim/rootfs"
+        echo "This need binfmt to be configured"
+        echo "  docker run --rm --privileged multiarch/qemu-user-static:register --reset."
+        ls -l $(pwd)/bin
+        ls -l $(pwd)/bin/busybox
+        $(pwd)/bin/busybox --help
+        $(pwd)/bin/busybox --list-modules
```
Comment (junaruga) [inline update.sh:52]: There is an interesting error here. `$(pwd)/bin/busybox --help` is not recognized

https://travis-ci.org/multiarch/busybox/builds/543095257

```
++pwd
+ls -l /home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox
-rwxr-xr-x 1 travis travis 2382376 Apr  1 05:17 /home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox
++pwd
+/home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox --help
./update.sh: line 51: /home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox: No such file or directory
```

This issue only happens on Travis CI environment.
This is okay on my local environment (Fedora 30).

**Comment by lafin** [comment]
To prevent accidental merging, you can use the Draft mode for PR.
https://github.blog/2019-02-14-introducing-draft-pull-requests/

**Comment by junaruga** [comment]
@lafin thank you for the information. Next time I will take care of using the draft mode, to prevent it to be merged wrongly. It seems that once we created the pull-request, can not change it to draft pull-request mode.

I am sorry I did not mention why I shared this working in progress pull-request (WIP).
Because I wanted to share the above issue "bin/busybox: No such file or directory" that I could not solve at that time. I expected if I shared publicly, I might get some advises.

**Comment by junaruga** [comment]
I would close this ticket to prevent it will be merged.
When I needed in the future, I will open new PR with draft mode if it is needed.

**Pull request closed without merging**